src/odemis/gui/util/img.py

```diff
@@ -1037,6 +1037,11 @@ def calculate_ticks(value_range, client_size, orientation, tick_spacing):
     while cur_val < max_val:
         tick_values.append(cur_val)
         cur_val += value_step
+    if len(tick_values) < 2:
```

Comment:
This line should check this:
`if len(tick_values) < num_ticks:`

Reply:
Yes, ideally. However, this is a simpler trick which just ensure that if something went wrong, at least we always show 2 ticks. So we compare to the constant 2.